### DISCUSSION TITLE
Master no longer building after once building bottom_navigation

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#11813] [Q&A] [answered]
Some days ago I checked out the PR for bottom navigation.
Now I simply went back and checked out `master` and doublechecked my repo is aligned with upstream/master.

However since then I am no longer able to build the app in AS with the following error:
![image](https://user-images.githubusercontent.com/949669/136656321-5030d6ce-9558-423e-8013-f5e19304dfbc.png)

Tried "Clean Project" and "Rebuild project" but it did not help.
Can build other branches (like `release` or `bottom_navigation`) without a problem. 
Any idea how to get rid of that and what caused it?

**Comment by fm-sys** [comment, marked as answer]
I have exactly the same problem. IMHO, this must be a gradle bug. Currently, I have added a resource file with name `bottom_navigation_menu.xml` and empty content (see below) to the menu folder. After one successful build you can remove that file again IIRC

```
<?xml version="1.0" encoding="utf-8"?>
<menu xmlns:android="http://schemas.android.com/apk/res/android">

</menu>

**Reply by Lineflyer** [reply]
Thanks, that got me going again.
But I need to keep the file, otherwise the problem returns... bit of a problem as I might accidentally commit this change :(

**Comment by Lineflyer** [comment]
Seems I solved whatever was the culprit by either of or a combination of the following steps:
- Clear /main/build
- Clear /build-cache
- Clear files in my systems `%temp%`
- Open another project and build it, then change back to cgeo

-
